### ISSUE TITLE
unexpected JSON parsing error

### ISSUE THREAD

**Issue by alec-djinn** [Ride-The-Lightning/RTL#929]
The following unexpected behavior occurred after having updated RTL to 0.12.0

The server starts regularly but it fails after logging in with
```
Error 500
Http failure response for http://localhost:3000/rtl/api/lnd/getinfo: 0 unknown error 
```

The server crashes immediately after.

Here is the log on terminal.



```
$ node rtl

[1/15/2022, 5:45:40 PM] DEBUG: App => Starting Express Application..

[1/15/2022, 5:45:40 PM] DEBUG: Config => Node Information: {"index":1,"ln_node":"Node 1","ln_implementation":"LND","macaroon_path":"/media/john/6TB/LND/lnd/lnd/data/chain/bitcoin/mainnet","config_path":"/media/john/6TB/LND/lnd/Lnd/lnd.conf","ln_server_url":"https://localhost:8080","user_persona":"OPERATOR","theme_mode":"NIGHT","theme_color":"PURPLE","log_level":"ERROR","fiat_conversion":true,"currency_unit":"USD","swap_server_url":"http://localhost:8081","swap_macaroon_path":"","boltz_server_url":"","boltz_macaroon_path":"","enable_offers":false,"bitcoind_config_path":"","channel_backup_path":"/media/john/6TB/LND/backup","log_file":"/home/john/Desktop/RTL/logs/RTL-Node-1.log"}

[1/15/2022, 5:45:40 PM] DEBUG: CORS => Setting up CORS..

[1/15/2022, 5:45:40 PM] DEBUG: CSRF => Setting up CSRF..

[1/15/2022, 5:45:40 PM] INFO: RTL => Server is up and running, please open the UI at http://localhost:3000.

[1/15/2022, 5:45:54 PM] INFO: RTLConf => Getting Initial RTL Configuration...

[1/15/2022, 5:45:54 PM] INFO: RTLConf => Initial RTL Configuration Received.

[1/15/2022, 5:45:54 PM] INFO: RTLConf => Updating Selected Node...
undefined:1
{"nc_sameSiteCookielax":"true","nc_sameSiteCookiestrict":"true","__Host-nc_sameSiteCookielax":"true","__Host-nc_sameSiteCookiestrict":"true","_xsrf":"2|b8781821|62d9d0c52dee50de1d9a0dc8f4e42eb0|1641681900","username-localhost-8888":""2|1:0|10:1642180148|23:username-localhost-8888|44:ZDU4YWQ2MjYxZGZlNDcwMjgwOGQ5M2M3MGY3ZjdkZDM":"|40cff8083ad4b8748c84f82bf3f333a7ad6b07b1139a8455beb101ad14d32fe8"","_csrf":"FbPDI5i-Q9v5omeOJsbf2z0U","XSRF-TOKEN":"0SmiyIPI-GOhgqYUHjp7qvRyujAkeA_kN-Yc","connect.sid":"s%3ARNFO6RpvrVVkb8rHqno2Z3rg3-to4J-q.DlTUPdCtloq6TL4HTXrQDoiZYKLeDyCnh4o6rSH2AE8"}
                                                                                                                                                                                                                                          ^

SyntaxError: Unexpected number in JSON at position 234
    at JSON.parse (<anonymous>)
    at file:///home/john/Desktop/RTL/backend/utils/authCheck.js:33:46
    at /home/john/Desktop/RTL/node_modules/jsonwebtoken/verify.js:223:12
    at getSecret (/home/john/Desktop/RTL/node_modules/jsonwebtoken/verify.js:90:14)
    at Object.module.exports [as verify] (/home/john/Desktop/RTL/node_modules/jsonwebtoken/verify.js:94:10)
    at Object.verifyWSUser [as verifyClient] (file:///home/john/Desktop/RTL/backend/utils/authCheck.js:26:13)
    at WebSocketServer.handleUpgrade (/home/john/Desktop/RTL/node_modules/ws/lib/websocket-server.js:258:22)
    at Server.<anonymous> (file:///home/john/Desktop/RTL/backend/utils/webSocketServer.js:45:38)
    at Server.emit (node:events:390:28)
    at onParserExecuteCommon (node:_http_server:725:14)
    at onParserExecute (node:_http_server:638:3)

Node.js v17.3.1
```

**Comment by ShahanaFarooqui** [comment]
@alec-djinn can you try to clear your browser’s cookies or run it with incognito mode. This bug was raised in our telegram group and has been fixed already. The fix will be available with v0.12.1 releasing today.

**Comment by ShahanaFarooqui** [comment]
Closing with commit `7a7f41c4465ec610f3ec6e2362c6abc8a2b49772`.

**Comment by alec-djinn** [comment]
> @alec-djinn can you try to clear your browser’s cookies or run it with incognito mode. This bug was raised in our telegram group and has been fixed already. The fix will be available with v0.12.1 releasing today.

Running it in incognito mode worked. Thanks.

**Comment by ShahanaFarooqui** [comment]
The error on cookie clearing is also fixed already, will be released soon. Thanks for the confirmation.
